docs/code.rst

```diff
@@ -119,6 +119,8 @@ in all templates
 Variable                            Description
 =================================   ====================================
 ``{{ ds }}``                        the execution date as ``YYYY-MM-DD``
+``{{ prev_ds }}``                   the previous execution date as ``YYYY-MM-DD``
+``{{ next_ds }}``                   the next execution date as ``YYYY-MM-DD``
```

Comment:
I think calling this the "next" execution date is sort of misleading. For example, if "this" execution is Monday, I'd expect the `prev_ds` to be Sunday's date, and `next_ds` to be Tuesday's date.

I wonder if we actually need `next_ds` at all, since it's equivalent to `ds`.

Reply:
Oh wait, maybe that's actually exactly what's intended? In that case, ignore this comment.

Reply:
That's right. `next_ds`=`ds` + `schedul_interval` in `YYYY-MM-DD` format.

Reply:
Got it, thanks.